### PR TITLE
Updated Vault API

### PR DESCRIPTION
**Pull request by jaypatel512**
- Added `total_required` as a parameter for list query.

### CODE REVIEW

**Comment by jaypatel512** [comment]
Shouldnt matter, as it defaults to true. However, added it.

File: rest-api-sdk/src/main/java/com/paypal/api/payments/CreditCard.java [hunk 1]
```diff
@@ -581,7 +581,8 @@ public static CreditCardHistory list(String accessToken) throws PayPalRESTExcept
 		parameters.put("page_size", "10");
 		parameters.put("sort_order", "asc");
 		parameters.put("sort_by", "create_time");
-		
+		parameters.put("total_required", "true");
```
Comment (juwlee) [inline rest-api-sdk/src/main/java/com/paypal/api/payments/CreditCard.java]: Is the default value `true`?

Comment (jaypatel512) [inline rest-api-sdk/src/main/java/com/paypal/api/payments/CreditCard.java]: Yes. As per the docs.

**Comment by jaypatel512** [comment]
Merged. Thanks @juwlee and @braebot for reviewing.

**Pull request merged**